tests/library/gemm_test.py

```diff
@@ -14,6 +14,22 @@
 N = dace.symbol('N')
 
 
+@pytest.mark.parametrize(('implementation', ),
+                         [('pure', ), ('MKL', ),
+                          pytest.param('cuBLAS', marks=pytest.mark.gpu)])
+def test_gemm_no_c(implementation):
+    Gemm.default_implementation = implementation
```

Comment:
please keep the old implementation and set it back. pytest does not create new python scripts when running.
Also, not everyone has MKL so you need to either mark it as such or have a try/except block